app-office/drawio-desktop-bin/metadata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>foti.giuseppe@gmail.com</email>
+		<name>Giuseppe Foti</name>
+	</maintainer>
+	<maintainer type="person" proxied="yes">
+		<email>zdanevich.vitaly@ya.ru</email>
+		<name>Vitaly Zdanevich</name>
```

Comment:
Did Vitaly agree to be added?

Since he is not yet a proxied maintainer in Gentoo, I would suggest to drop him for now and add him once he contributed to this package in ::gentoo.

Reply:
I talked with him and he agree to be included as a maintaner

Reply:
Ok @Flowdalic, I will remove him from metadata to include him later if he will contribute on this ebuild.